projects/step-frontend/src/lib/modules/timeseries/ranger/ts-ranger.component.ts

```diff
@@ -88,7 +95,9 @@ export class TSRangerComponent implements OnInit, AfterViewInit, OnChanges {
 
   selectRange(fromTimestamp?: number, toTimestamp?: number) {
     let select = this.transformRangeToSelect({ from: fromTimestamp, to: toTimestamp });
-
+    // select!.width = select!.width! + 100;
+    // console.log(this.getSize(), select);
+    console.log(this.uplot.bbox, select);
```

Comment:
I don't want to nag you all the time about debug / commented out code, but if we remove it in the PRs we have a much easier job with keeping the master clean.

Reply:
my fault. no worries. I will try to get used to check for console.logs before pushing